openmetadata-ui/src/main/resources/ui/src/utils/DataAssetsHeader.utils.tsx

```diff
@@ -178,13 +180,15 @@ export const getDataAssetsHeaderInfo = (
           )}
           {mlModelDetail.server && (
             <ExtraInfoLink
+              newTab
               href={mlModelDetail.server}
               label={t('label.server')}
               value={mlModelDetail.server}
             />
           )}
           {mlModelDetail.dashboard && (
             <ExtraInfoLink
+              newTab
```

Comment:
@fuzmish, I believe this is an internal path, so we don’t need the newTab as true here.

Reply:
Thank you for your comment. a4bd542 I've fixed it.